fabulaws-config.yml

```diff
@@ -178,14 +178,14 @@
   volume_sizes:
     production:
       cache: 10
-      db-master: 100
-      db-slave: 100
+      db-master: 250
+      db-slave: 250
       web: 10
       worker: 50
     testing:
       cache: 10
-      db-master: 100
-      db-slave: 100
+      db-master: 250
+      db-slave: 250
       web: 10
```

Comment:
while we're here, should we bump up the web size a bit, since they're currently 70% full?